### ISSUE TITLE
[app flow] ejs fIles are not copied to statics directory in dev mode

### ISSUE THREAD

**Issue by maxaon** [wix/yoshi#1218]
# 🐛 Bug Report

### 😯 Current Behavior
If using regular flow all ejs files are copied to dist/statics. 
If **projectType** is **app** then ejs files are not watched and not copied to dist/statics. And app fall on first request.

...

### 🤔 Expected Behavior

During run ejs files should be copied to dist

### 💻 Code Sample

```javascript
   const templatePath = path.join(
      hadron.staticLocalPath(artifactId, './dist/statics'),
      'index.ejs',
    );
    const html = await renderEjsFile(templatePath, renderModel, {
      cache: isProduction,
      filename: templatePath,
    });

```
### 🌍 Your Environment

```
  System:
    OS: Linux 4.15 Ubuntu 18.04.2 LTS (Bionic Beaver)
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
  Binaries:
    Node: 10.15.3 - /usr/bin/node
    Yarn: 1.15.2 - /usr/bin/yarn
    npm: 6.8.0 - ~/.npm-global/bin/npm
    Watchman: Not Found
  Browsers:
    Chrome: 73.0.3683.86
    Firefox: 66.0.2
  npmPackages:
    @storybook/core: Not Found
    @storybook/react: Not Found
    babel-preset-yoshi: Not Found
    eslint-config-yoshi: Not Found
    eslint-config-yoshi-base: Not Found
    jest-environment-yoshi-bootstrap: Not Found
    jest-environment-yoshi-puppeteer: Not Found
    jest-yoshi-preset: Not Found
    storybook: Not Found
    tslint-config-yoshi: Not Found
    tslint-config-yoshi-base: Not Found
    typescript: Not Found
    webpack: Not Found
    yoshi-config: Not Found
    yoshi-helpers: Not Found
    yoshi-runtime: Not Found

 Yoshi config
{
  "projectType": "app",
  "entry": {
    "videoMakerModal": "./videoMakerModal",
    "videoMakerModalCritical": "./videoMakerModal.critical"
  },
  "externals": {
    "react": "React",
    "react-dom": "ReactDOM",
    "react-intl": "ReactIntl",
    "prop-types": "PropTypes",
    "bluebird": "Promise"
  },
  "servers": {
    "cdn": {
      "port": 3220
    }
  }
}
```

**Comment by ronami** [comment]
Thanks for the issue, created a fix for it in #1220.

**Comment by ranyitz** [comment]
@maxaon You don't need to handle hadron yourself, as you have https://github.com/wix-private/fed-infra/tree/master/wix-bootstrap-renderer

If you choose to handle it yourself you should default to `./src` rather than `./dist/static`

Just to be clear:
```diff
-  hadron.staticLocalPath(artifactId, './dist/statics')
+  hadron.staticLocalPath(artifactId, './src')
```

Closing
